.cookiecutter/includes/tox/setenv

```diff
@@ -0,0 +1,7 @@
+MB_DB_USER = {env:MB_DB_USER:postgres}
+# As it's Postgres itself that will be contacting these DBs, this will
+# happen inside docker, so we use docker names here
+H_US_DATABASE_URL = {env:H_US_DATABASE_URL:postgresql://postgres:postgres@h_postgres_1:5432/postgres}
+H_CA_DATABASE_URL = {env:H_CA_DATABASE_URL:postgresql://postgres:postgres@h_postgres_1:5432/postgres}
+LMS_US_DATABASE_URL = {env:LMS_US_DATABASE_URL:postgresql://postgres:postgres@lms_postgres_1:5432/postgres}
+LMS_CA_DATABASE_URL = {env:LMS_CA_DATABASE_URL:postgresql://postgres:postgres@lms_postgres_1:5432/postgres}
```

Comment:
These are the custom envvars that are [currently in `setenv`](https://github.com/hypothesis/report/blob/f309dde299884d2e6370f96fda896c6ea7f5d069/tox.ini#L15-L21), the cookiecutter requires you to put them in an include